stable/hadoop/templates/configmaps/hadoop-config.yaml

```diff
@@ -0,0 +1,296 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-config
```

Comment:
`name: {{ template "hadoop.fullname" . }}`

Only append suffix for components. Fix references below.